src/pages/explorer/Explorer.jsx

```diff
@@ -13,141 +13,235 @@ import { Controlled as CodeMirror } from 'react-codemirror2';
-              <Icon type="info-circle" style={{ color: 'rgba(0,0,0,.45)' }} />
-            </Tooltip>
+      <div className='flex-box'>
+        <Sidenav selectedItem='explorer' />
+        <div className='page-content'>
+          <div className='header-flex'>
+            <Header name='Explorer' color='#000' fontSize='22px' />
+            <Button.Group>
+              <Button
+                type={props.selectedIDE === 'graphql' ? 'primary' : 'default'}
+                onClick={() => props.setSelectedIDE('graphql')}
+              >
+                GraphQL
+              </Button>
+              <Button
+                type={props.selectedIDE === 'spaceapi' ? 'primary' : 'default'}
+                onClick={() => props.setSelectedIDE('spaceapi')}
+              >
+                Space API
+              </Button>
+            </Button.Group>
           </div>
-          {!useAdminToken && <div className="row">
-            <Input.Password placeholder="Token to authorize request" value={token} onChange={e => setToken(e.target.value)} />
-          </div>}
-          <div className="row">
-            <CodeMirror
-              value={value}
-              options={{
-                mode: { name: "javascript", json: true },
-                lineNumbers: true,
-                styleActiveLine: true,
-                matchBrackets: true,
-                autoCloseBrackets: true,
-                tabSize: 2,
-                autofocus: true
-              }}
-              onBeforeChange={(editor, data, value) => {
-                setValue(value)
-              }}
-            />
-          </div>
-          <div className="row apply-container">
-            <Select value={selectedTemplate} style={{ minWidth: "240px" }} showSerach={true} onChange={setSelectedTemplate} placeholder="Pick a Template">
-              <Option value="insert">Insert Document</Option>
-              <Option value="get">Get documents</Option>
-              <Option value="call">Call a function</Option>
-            </Select>
-            <Button type="primary" onClick={applyRequest} loading={loading}>
-              Apply
-          </Button>
-          </div>
-          {loading === false && response && <div className="row">
-            <h3>Status: <Tag color={response.status === 200 ? "#4F8A10" : "#D8000C"}>{response.status}</Tag></h3>
-            <h3>Result: </h3>
-            <div className="result">
-              <CodeMirror
-                value={JSON.stringify(response.data, null, 2)}
-                options={{
-                  mode: { name: "javascript", json: true },
-                  tabSize: 2,
-                  readOnly: true
-                }}
-              />
+          {props.selectedIDE === 'graphql' && (
+            <div className='graphql'>
+              <GraphiQL fetcher={(graphQLParams) => graphQLFetcher(graphQLParams, props.match.params.projectId)} />
```

Comment:
Pass a `schema` prop to the `GraphiQL` component whose value will be `null` in order to avoid the error that we see in the response box.